pkgs/by-name/wh/whatsie/package.nix

```
@@ -0,0 +1,54 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  libX11,
+  libxcb,
+  qt5,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "whatsie";
+  version = "4.15.3";
+
+  src = fetchFromGitHub {
+    owner = "keshavbhatt";
+    repo = "whatsie";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-fbbyh8Q1jbKY+4ZEexDvIrhdcP+uCYAnhtUQsSqS0jw=";
+  };
+  sourceRoot = "${finalAttrs.src.name}/src";
+
+  buildInputs = [
+    libX11
+    libxcb
+    qt5.qtwebengine
+  ];
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    qt5.qmake
+  ];
+
```

Please test if you can compile this package with this stricter setting:

```suggestion

  strictDeps = true;

```

If it fails, explicitly set as false.

Roughly speaking, this setting enables some stricter flags that are helpful for cross compilation and a better separation of the multiple phases of building.

---

`strictDeps = true` fails with `> Project ERROR: Could not find feature webengine-spellchecker.` - which is interesting, because it's the same error I get if I don't include `qt5.qtwebengine`. But despite of including either `qt5.qtwebengine` or `qt5.full` - build with `strictDeps = true` fails.

---

Interesting. Usually it indicates we should fill the `buildInputs` and `nativeBuildInputs` with more accurate packages.

But let's not go down that rabbit hole for now.